.gitlab-ci.yml

```diff
@@ -2,8 +2,6 @@ Python 3 POCL:
   script: |
     export PY_EXE=python3
     export PYOPENCL_TEST=portable:pthread
-    # cython is here because pytential (for now, for TS) depends on it
-    export EXTRA_INSTALL="pybind11 cython numpy mako mpi4py oct2py"
```

Comment:
These seem to have been copy pasted from [meshmode/.gitlab-ci.yml](https://github.com/inducer/meshmode/blob/main/.gitlab-ci.yml), but they're not needed here, from what I can tell.